include/caffe/net.hpp

```diff
@@ -25,6 +25,7 @@ class Net {
  public:
   explicit Net(const NetParameter& param, const Net* root_net = NULL);
   explicit Net(const string& param_file, Phase phase,
+      const int level = 0, const vector<string>* stages = NULL,
```

Comment:
As per Evan's comments below, I reverted back to the original behavior of moving `root_net` to the end.